cpp/csp/adapters/websockets/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+set(WS_CLIENT_HEADER_FILES
+        ClientAdapterManager.h
+        ClientInputAdapter.h
+        ClientOutputAdapter.h
+        ClientHeaderUpdateAdapter.h
+        WebsocketEndpoint.h
+)
+
+set(WS_CLIENT_SOURCE_FILES
+        ClientAdapterManager.cpp
+        ClientInputAdapter.cpp
+        ClientOutputAdapter.cpp
+        ClientHeaderUpdateAdapter.cpp
+        WebsocketEndpoint.cpp
+        ${WS_CLIENT_HEADER_FILES}
+)
+
+add_library(csp_websocket_client_adapter STATIC ${WS_CLIENT_SOURCE_FILES})
+set_target_properties(csp_websocket_client_adapter PROPERTIES PUBLIC_HEADER "${WS_CLIENT_SOURCE_FILES}")
+
+find_package(websocketpp REQUIRED)
+set(OPENSSL_USE_STATIC_LIBS TRUE)
```

Comment:
Is this strictly required?

Reply:
I guess it doesn't explicitly need to search for the static libs